app/routes/__public/oauth/$id.tsx

```diff
@@ -22,6 +22,17 @@ import AES from "crypto-js/aes";
 
 const OauthConnectionPage = (): React.ReactElement => {
   const data = useLoaderData<typeof loadData>();
+
+  const postMessage = () => {
+    if (window.opener && !window.opener.closed && "body" in data) {
+      window.opener.postMessage(data.body.accessToken, "*");
```

Comment:
this is so sick!

Reply:
what does the asterisk do?

Reply:
`targetOrigin`
kept as asterisk because I figured this could be coming from many places (any SamePage app)